playbooks/deployments.md

```diff
@@ -26,142 +26,64 @@ description: How systems are deployed at Artsy
+convenient reuse. See
+[Convection's .circleci/config.yml](https://github.com/artsy/convection/blob/master/.circleci/config.yml) for a
+complete example.
+
+We use [horizon](https://github.com/artsy/horizon/) to poll for the status of our deployments and visualize them as
+a dashboard. See [its instructions](https://github.com/artsy/horizon#add-a-new-project) for defining a new project
+and configuring release pull requests to be opened [and optionally merged] automatically.
+
+If a project should _not_ be released for any reason (such as needing QA or known issues in staging),
+[create a deploy block](https://github.com/artsy/horizon#add-a-deploy-block) to record the reason and timing. The
+[artsy/release](https://github.com/artsy/orbs/blob/master/src/release/release.yml) orb defines a `block` step that
+will respect any unresolved deploy blocks and cause release builds to short-circuit.
+
+If a project strays from these common practices, it's _especially_ important to document the correct process in its
+README.
+
+## Setting up a project in CircleCI
+
+- See [CircleCI's docs](https://circleci.com/docs/2.0/getting-started/#section=getting-started) for general set-up
+  steps.
+- Artsy's project templates depend on write access to the repo (in order to push branches from CI steps). To create
+  a _read+write Github key_ for CircleCI (rather than the default read-only):
+  - Generate a key with a helpful label: `ssh-keygen -t rsa -b 4096 -m PEM -C "github_rw_key_for_circle"` (provide
+    a blank passphrase).
+  - Log into Github as the `artsyit` user and, in the project's settings, go to _Deploy keys_ > _Add deploy key_.
+    Give the key a descriptive name (like the label above) and paste in the contents of the _public_ (`.pub`) key
+    file.
+  - Check the _Allow write access_ box and click the _Add key_ button to save the new key.
+  - In the CircleCI project settings, go to _SSH Permissions_ > _Add SSH Key_.
+  - Enter `github.com` for _Hostname_ and the contents of the _private_ key file for _Private Key_, then click _Add
+    SSH Key_ to save.
+- In the projects' settings, generally speaking:
+  - _build forked pull requests_ should be enabled
+  - _pass secrets to builds from forked pull requests_ should be disabled
+
+## "Hot" fixes and roll-backs
+
+Sometimes there are urgent fixes to release. We always prefer using the typical pipeline to "roll forward," even in
+cases of reverted PRs or timely fixes. When it's critical to avoid the delay of a full CI run or staging deploy,
+it's possible to simply `hokusai production deploy <tag>` where _tag_ is an image tag (including git SHAs) and can
+refer to a previous release such as `production--2018-09-25--10-19-2`. (Run `hokusai registry images` to see recent
+tags.) Projects that depend on other release operations (e.g., to compile assets or update configuration) may
+require additional steps.
```

Comment:
Thinking good to stress here that this will deploy the docker image for the project (i.e. the code and dependencies as an artifact) but will _not_ include orchestration (changes to Kubernetes yaml files) as well as any release operations.

Somewhat tangential @joeyAghion do we want to deprecate `hokusai [staging|production] update`?

Reply:
That's a good point [that I didn't realize] about the orchestration.

I've used the `... update` command recently, either with `--dry-run` to validate some in progress changes, or during incidents. From my perspective it would be good to keep, even as an "advanced" command that most folks don't end up needing to run.